Composer/packages/ui-plugins/virtual-assistant-creation/src/models/creationOptions.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export enum AvailablePersonalities {
```

Comment:
Prefer string unions to enums
@hatpick - what are the specific problem with enums again?

Reply:
lol, 1. union types makes debugging much easier, 2. union types dont need importing.